src/openeo_processes/cubes.py

```diff
@@ -263,12 +290,26 @@ def exec_xar(data, output_filepath='out', format='GTiff', options={}):
             if hasattr(data, 'time') and hasattr(data.time, 'units'):
                 data.time.attrs.pop('units', None)
             # end workaround
+            
+            data = refactor_data(data)
             data.to_netcdf(path=output_filepath)
-        elif format == 'GTiff':
+
+        elif format.lower() in ['gtiff','gtif','geotiff','geotif','tiff','tif']:
```

Comment:
```suggestion
        elif format.lower() in ['gtiff','geotiff']:
```

GTiff is the only allowed option if following the API spec closely. I could see that people type geotiff by accident, but I wouldn't allow too many options here for interoperability reasons.

Reply:
Ok fine, I'll keep only the two options as you suggest